pkg/suggestion/NASenvelopenet/generate.py

```diff
@@ -0,0 +1,487 @@
+import sys
+import json
+import copy
+import re
+import random
+import math
+import argparse
+from collections import Counter
+
+
+class NACAlg:
+    def __init__(self, algorithm, config, stages, initcell):
+        self.algorithm =algorithm
+        self.config = config
+        print(self.config)
+        if self.algorithm == "envelopenet" or self.algorithm == "envelopenet2":
+            self.max_filter_prune = int(self.config["max_filter_prune"])
+            if 'worst_case' in self.config:
```

Comment:
There were some additional experiments performed like worst_case, randomized search, and etc. Their purpose was to perform an ablation study for the accompanying paper, so I guess those parts are not required in the final search algorithm and would also reduce unused code.